data/data/gcp/master/main.tf

```diff
@@ -3,7 +3,7 @@ resource "google_compute_instance" "master" {
 
   name         = "${var.cluster_id}-master-${count.index}"
   machine_type = var.machine_type
-  zone         = var.zones[count.index]
+  zone         = var.zones[count.index % length(var.zones)]
```

Comment:
you can use https://www.terraform.io/docs/configuration/functions/element.html instead of the `%`

Reply:
The % wraps back to zone[0] for instance[4]

Reply:
that's what `element` does too. `If the given index is greater than the length of the list then the index is "wrapped around" by taking the index modulo the length of the list:`

Reply:
I completely missed that. +1